### PR TITLE
Basin UI (3) - Yield info + misc fixes

### PR DESCRIPTION
**Pull request by Bean-Sama**
In This PR:

- Replaced Well Trading Fees column for Yield & added tooltip on hover
- Updated tooltips throughout the app
- modified Reserves component formatting
- added well reserve tooltip displaying well reserves & TWAReserves if the well has MultiFlowPump
- Disallow user from inputting an amount greater than their balance in 'Remove Liquidity'
- Fixed a bug where duplicated data would cause the chart to display nothing
- other minor housekeeping / formatting / typing along the way
- Fix charts Y data points overflowing chart heights

To deploy an empty well for testing, you can run npx ts-node projects/examples/src/wells/deployNewWell.ts from the project root folder

### CODE REVIEW

File: projects/dex-ui/src/wells/useBeanstalkSiloWhitelist.ts
Comment (0xalecks) [inline projects/dex-ui/src/wells/useBeanstalkSiloWhitelist.ts:4]: Would be better to extract hard coded addresses to a central, easily to find, file like 'addresses.tx' or so..

**Review by 0xalecks — CHANGES_REQUESTED** [review]

File: projects/dex-ui/src/wells/apyFetcher.ts
Comment (Bean-Sama) [reply projects/dex-ui/src/wells/apyFetcher.ts:14]: @hellofromguy should we hold off on the APYs until BIP39 gets deployed?

Comment (spirit-of-beanthoven) [reply projects/dex-ui/src/wells/apyFetcher.ts:14]: no

Comment (hellofromguy) [reply projects/dex-ui/src/wells/apyFetcher.ts:14]: no

Comment (hellofromguy) [reply projects/dex-ui/src/wells/apyFetcher.ts:14]: should be able to swap it out with the new APY data in the subgraph when the time comes

File: projects/dex-ui/src/wells/useBeanstalkSiloWhitelist.ts
Comment (0xalecks) [inline projects/dex-ui/src/wells/useBeanstalkSiloWhitelist.ts:21]: In order for .getSeeds() to work, you MUST call sdk.refresh()

See how we do it in the UI:
https://github.com/AgrarianAlliance/Beanstalk/pull/45/files#diff-c46de2fb1f0f47ec5cdb8a6c77a955c9ec346362f812fb70e4a38ee0549645e1R71

Note, this requires BIP39, which hasn't been deployed yet so this means we cannot merge this PR until after.

Happy to chat about this if you have questions. You don't need to set the seed for each token like I did in the UI PR below the line commented, that was only to update the UI's tokens... the SDK will update its tokens on refresh().

**Review by 0xalecks — CHANGES_REQUESTED** [review]

**Pull request merged**
